src/ext/issuance.js

```diff
@@ -41,6 +46,25 @@ function signReqCF(url) {
     return xhrInfo;
 }
 
+function signReqHC(url) {
+    let reqUrl = url.href;
+    const isIssuerUrl = ACTIVE_CONFIG["issue-action"]["urls"]
+        .map(issuerUrl => patternToRegExp(issuerUrl))
+        .filter(re => reqUrl.match(re)).length > 0;
+
+    if (!isIssuerUrl) {
+        return null;
+    }
+
+    sentTokens[reqUrl] = true;
+    // Generate tokens and create a JSON request for signing
+    let tokens = GenerateNewTokens(TOKENS_PER_REQUEST);
+    const request = BuildIssueRequest(tokens);
+    // Construct info for xhr signing request
+    let xhrInfo = {newUrl: reqUrl, requestBody: `blinded-tokens=${request}&captcha-bypass=true`, tokens: tokens};
```

Comment:
Is adding `&captcha-bypass=true` to the request body here intentional? In the CF case we add this to the URL as a query parameter.

Reply:
Yes, due to the way our routes are set up at the moment appending it to the URL ends up with some messy handling, so we opted for the body instead.